_socketsGenerator/test/createSockets.test.js

```diff
@@ -0,0 +1,169 @@
+import sinon from 'sinon';
+import { expect } from 'chai';
+import functions from '../createSockets';
+
+const { 
+  renderEndpoints,
+  createDirs,
+  createYamlInput,
+  createScript,
+  whiteListMachine,
+  createSockets 
+} = functions;
+
+describe('Test createSockets.js', function () {
+
+  describe('renderEndpoints', function () {
+    let array = [];
+    beforeEach(function() {
```

Comment:
be sure to check if beforeEach is nessesary, as it results in longer tests, try checking before settling for one result